includes/analytics/class-analytics.php

```diff
@@ -86,6 +86,7 @@ public static function get_prophecy_token_header( $args = [] ) {
 		$site_name    = get_bloginfo( 'name' );
 
 		$defaults = [
+			'date_time'       => current_time( 'mysql' ),
```

Comment:
We don't need to explicitly pass the date_time because Mixpanel (and any other analytic software should) automatically handles this.